app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java

```diff
@@ -138,7 +138,7 @@ public enum SettingsEnum {
     HIDE_SHORTS_COMMENTS_BUTTON("revanced_hide_shorts_comments_button", BOOLEAN, FALSE),
     HIDE_SHORTS_REMIX_BUTTON("revanced_hide_shorts_remix_button", BOOLEAN, FALSE),
     HIDE_SHORTS_SHARE_BUTTON("revanced_hide_shorts_share_button", BOOLEAN, FALSE),
-    HIDE_SHORTS_INFO_PANEL("revanced_hide_shorts_info_panel", BOOLEAN, FALSE),
+    HIDE_SHORTS_INFO_PANEL("revanced_hide_shorts_info_panel", BOOLEAN, TRUE),
```

Comment:
I personally think, most people would also want other shorts components to be hidden by default such as the remix or join component as they are almost not used in most cases. The main reason everything is set to false was motivated by the previously mentioned issue. Setting this to true causes that issue. If we want to ignore the issue for now, I think we can hide remix, join & thanks by default to (in another PR).